tests/openai/test_parallel.py

```diff
@@ -14,6 +14,23 @@ class GoogleSearch(BaseModel):
     query: str
 
 
+def test_sync_parallel_tools__error(client):
+    client = instructor.patch(client, mode=instructor.Mode.PARALLEL_TOOLS)
+
+    with pytest.raises(TypeError):
```

Comment:
@lukevs We should definitely be switching back to regular tools mode if we can detect that it's not an iterable.

Reply:
Need it still for the `Boy | Girl` case too right (to get the name disambiguation)